docs/DeveloperManuals/PluginImplementation.md

```diff
@@ -73,7 +71,7 @@ There's a lot of information in the diagram but we don't expect you to digest it
 
 In this guide, we'll walk through how to create a data source plugin from scratch. 
 
-The example in this tutorial comes from DevLake's own needs of managing [CLAs](https://en.wikipedia.org/wiki/Contributor_License_Agreement). Whenever DevLake receives a new PR on GitHub, we need to check if the author has signed a CLA by referencing `https://people.apache.org/public/icla-info.json`. This guide will demonstrate how to collect the ICLA info from Apache API, cache the raw response, and extract the raw data into a relational table ready to be queried.
+Apache requires the project to confirm whether the Contributors and Committers of the project have signed the CLA. So we need to check whether committers have signed the CLA by requesting Apache API and whether contributors have signed by requesting the mailing list. Here we will explain how to request and cache committer information from the Apache API and extract the structured data. There is just an introduction for contributors' at the end of this article.
```

Comment:
In this section, we will describe how to create a data collection plugin from scratch. The data to be collected is information about all Committers and Contributors of the Apache project, in order to check whether they have signed the CLA.
- requesting `https://people.apache.org/public/icla-info.json` to get the Committers' information
- requesting the `mailing list` to get the Contributors' information
We will focus on demonstrating how to request and cache information about all Committers through the Apache API and extract structured data from it. The collection of Contributors will only be briefly described.